nodehid.js

```diff
@@ -102,6 +111,42 @@ HID.prototype.resume = function resume() {
     }
 };
 
+HID.prototype.write = function write(data) {
+    if (this._closed) {
+		throw new Error('Unable to write to a closed HID device');
+	}
+
+	return this._raw.write(toBuffer(data));
+};
+
+HID.prototype.sendFeatureReport = function sendFeatureReport(data) {
+	if (this._closed) {
+		throw new Error('Unable to sendFeatureReport to a closed HID device');
+	}
+
+	return this._raw.sendFeatureReport(toBuffer(data));
+};
+
+// utility method to convert an Array or Buffer to a Buffer for the native C++
+// code to work with.
+function toBuffer (data) {
```

Comment:
@todbot `toBuffer()` is how this pull request maintains backward compatibility with clients expecting to be able to send an array of bytes. The array is detected and explicitly converted to a buffer (which keeps the native code much cleaner).